src/app/modules/core/services/wallet.service.ts

```diff
@@ -94,4 +94,8 @@ export class WalletService {
   recover(request: RecoverWalletRequest): Observable<any> {
     return this.http.post(`${this.apiUrl}/wallet/recover`, request);
   }
+
+  exitAccounts(request: AccountVoluntaryExitRequest): Observable<any> {
+    return this.http.post(`${this.apiUrl}/wallet/exit-accoints`, request);
```

Comment:
typo: `accoints`

Reply:
Hey @olehyev138 I think this endpoint will be `${this.apiUrl}/accounts/exit` when I implement it in the backend

Reply:
I will update the code